sdcm/utils/log.py

```diff
@@ -164,7 +171,22 @@ def configure_logging(exception_handler=None,  # pylint: disable=too-many-argume
             },
             'anyconfig': {
                 'level': 'ERROR'
-            }
+            },
+            'argus': {
+                'handlers': ['argus'],
+                'level': 'DEBUG',
+                'propagate': False
+            },
+            'sdcm.argus_test_run': {
+                'handlers': ['argus'],
+                'level': 'DEBUG',
+                'propagate': False
+            },
+            'TestRun': {
```

Comment:
@k0machi why did you create a separate logger for `TestRun` (instead of using module logger)?

Reply:
per our conversation, I created a task to fix this in Argus (use argus logger)